src/governance/governance.h

```diff
@@ -103,9 +93,9 @@ class CRateCheckBuffer
         return nMax;
     }
 
-    int GetCount()
+    int GetCount() const
     {
-        int nCount = 0;
+        int nCount;
```

Comment:
Whats the reason for doing fun like this? Asking because i saw you recently doing the same in an other PR 🙈 IMO having a defined initial state of a variable is always better and nothing bad no matter if are all cases are covered because the code using this variable can change and undefined behaviour can be missed.

Reply:
I guess it would be even better to get rid of `nCount` completely and do smth like
```
if (nDataEnd > nDataStart) {
    return nDataEnd - nDataStart;
}

return RATE_BUFFER_SIZE - nDataStart + nDataEnd;
```

below

Reply:
It's just because it's redundant, every path writes something to nCount, but never accesses it, so ClangTidy gives me a warning for it.

See cb9c6721548206d7467c36ce43c5e7e031b9d2a0